src/components/SearchBar.tsx

```diff
@@ -306,7 +306,7 @@ export function SearchBar({
         onClick={handleSubmit}
       >
         {renderAutocompleteResult(
-          { value: result.query },
+          { value: result.query, inputIntents: [] },
```

Comment:
what is this?

Reply:
inputIntents was added to search core in this [PR](https://github.com/yext/search-core/pull/230), and released in v2.2.0. search-headless was previously using v2.1.0 of core which didn't include that change, and now its on v2.3.0 which includes the change